File: 3st/Hyeonseo/programmer/길이에 따른 연산.py
def solution(num_list):
    if len(num_list) >= 11:  # 리스트 길이가 11 이상이면
        return sum(num_list)
    else:  # 리스트 길이가 10 이하이면
        product = 1
        for num in num_list:
            product *= num
        return product

File: 4st/Jisoo/programmer/길이에 따른 연산.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 1;
        if (num_list.length >= 11) {
            for (int i = 0; i < num_list.length; i++) {
                answer += num_list[i];
            }
            answer = answer - 1;
        }
        else if (num_list.length < 11) {
            for (int i = 0; i < num_list.length; i++) {
                answer = answer * num_list[i];
            }
        }
        return answer;
    }
}